igcollect/ping.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+"""igcollect - Latency collection
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+import _thread
+import argparse
+import subprocess
+
+from time import time, sleep
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Ping colecctor script for Graphana ')
+    parser.add_argument('adress', metavar='URL', type=str, nargs='+',
+                        help='the address zu ping to')
+    parser.add_argument('-c', '--count', type=int, default=20,
+                        help='how many tims should the script tries to ping')
+    parser.add_argument('--prefix', default='ping',
+                        help='the path to the value in Graphana')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    hostname = args.adress
```

Comment:
it would be beneficial to keep the same name. decide on either `hostname` or `address`